contracts/margin/external/ERC20/ERC20Position.sol

```diff
@@ -442,14 +433,12 @@ contract ERC20Position is
         );
 
         // Destroy the margin tokens
-        delete balances[onBehalfOf];
```

Comment:
`delete` does the same thing as setting it to 0 right?

Reply:
/ costs the same amount of gas

Reply:
ye